lib/ansible/modules/cloud/amazon/ecs_taskdefinition.py

```diff
@@ -369,8 +372,7 @@ def _task_definition_matches(requested_volumes, requested_containers, existing_t
         if module.params['state'] == 'absent':
             if 'arn' in module.params and module.params['arn'] is not None:
                 task_to_describe = module.params['arn']
-            elif 'family' in module.params and module.params['family'] is not None and 'revision' in module.params and \
-                            module.params['revision'] is not None:
+            elif 'family' in module.params and module.params['family'] is not None and 'revision' in module.params and module.params['revision'] is not None:
```

Comment:
It seems like this could be `if module.params.get('family') is not None and module.params.get('revision') is not None`